editor/property_selector.h

```diff
@@ -48,6 +48,7 @@ class PropertySelector : public ConfirmationDialog {
 	void _update_search();
 	void _confirmed();
 	void _item_selected();
+	void _multi_item_selected(Object *p_item, int column,bool is_item_selected);
```

Comment:
```suggestion
	void _multi_item_selected(Object *p_item, int p_ column, bool p_is_item_selected);
```
Prefix arguments with `p_`, and style

Reply:
Fixed in latest commit